<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZZNRemark</class>
 <widget class="QMainWindow" name="ZZNRemark">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1001</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1001</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1001</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>yolo系列格式_数据标注软件</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/image/image/logo.png</normaloff>:/image/image/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>4</y>
      <width>281</width>
      <height>981</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>4</y>
      <width>1308</width>
      <height>981</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="foregroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1603</x>
      <y>10</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>标注类型</string>
    </property>
    <widget class="QComboBox" name="comboBoxType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelColorType">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxRect">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1603</x>
      <y>120</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>标注属性</string>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>151</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>宽度</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>高度</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>171</width>
       <height>151</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSpinBox" name="spinBoxX">
      <property name="maximum">
       <number>1267</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxY">
      <property name="maximum">
       <number>950</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxWidth">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1268</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxHeight">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>951</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionFront"/>
   <addaction name="actionBack"/>
   <addaction name="separator"/>
   <addaction name="actionOriginalSize"/>
   <addaction name="actionReduce"/>
   <addaction name="separator"/>
   <addaction name="actionDefautPath"/>
   <addaction name="actionSettingType"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/image/image/open.png</normaloff>:/image/image/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开图片</string>
   </property>
  </action>
  <action name="actionDefautPath">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/image/image/path.png</normaloff>:/image/image/path.png</iconset>
   </property>
   <property name="text">
    <string>默认路径</string>
   </property>
   <property name="toolTip">
    <string>设置默认路径</string>
   </property>
  </action>
  <action name="actionSettingType">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/image/image/colors.png</normaloff>:/image/image/colors.png</iconset>
   </property>
   <property name="text">
    <string>标注类型</string>
   </property>
   <property name="toolTip">
    <string>设置标注类型</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/image/image/delete.png</normaloff>:/image/image/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除选中图片</string>
   </property>
  </action>
  <action name="actionFront">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/image/image/front.png</normaloff>:/image/image/front.png</iconset>
   </property>
   <property name="text">
    <string>前置</string>
   </property>
   <property name="toolTip">
    <string>将选中的标注框前置</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/image/image/behind.png</normaloff>:/image/image/behind.png</iconset>
   </property>
   <property name="text">
    <string>后置</string>
   </property>
   <property name="toolTip">
    <string>将标注的选中框后置</string>
   </property>
  </action>
  <action name="actionReduce">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/image/image/reduce.png</normaloff>:/image/image/reduce.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小图像尺寸</string>
   </property>
  </action>
  <action name="actionOriginalSize">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/image/image/enlarge.png</normaloff>:/image/image/enlarge.png</iconset>
   </property>
   <property name="text">
    <string>1:1</string>
   </property>
   <property name="toolTip">
    <string>1:1放大图像尺寸</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/image/image/about.png</normaloff>:/image/image/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>版本信息</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
